codice/ddf
# DDF-4311 Remove UI component, add url functionality to the CLI, and u…

…pdate docs.

#### What does this PR do?
Removes the offline gazetteer from the UI, adds url functionality to the CLI, and updates the gazetteer documentation and  CLI help message to reflect the change.

*Edit:* @emmberk has added a fix for a related issue to this PR. The issue prevented loading geocode files.

#### Who is reviewing it? 
@mdang8 @kcover @peterhuffer @emmberk @oconnormi @stustison @troymohl 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@clockard

#### How should this be tested?

- Try to add geocodes through the command line by using a link. 
`gazetteer:update linkToResource`  Links can be found [here](http://download.geonames.org/export/dump/)

- In the admin UI, verify the GeoNames Configuration tab under the Spatial app, is no longer there.


#### What are the relevant tickets?
[DDF-4311](https://codice.atlassian.net/browse/DDF-4311)

#### Screenshots
Before:
<img width="382" alt="screen shot 2018-11-07 at 7 51 41 am" src="https://user-images.githubusercontent.com/12192559/48137906-136ea400-e271-11e8-9a76-df9f6feb1ebc.png">

After:
<img width="397" alt="screen shot 2018-11-07 at 8 29 27 am" src="https://user-images.githubusercontent.com/12192559/48137914-16699480-e271-11e8-8aa4-c2c3d8f59ba1.png">


#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
